app/models/manageiq/providers/ansible_tower/configuration_manager/ansible_tower_job.rb

```diff
@@ -0,0 +1,60 @@
+class ManageIQ::Providers::AnsibleTower::ConfigurationManager::AnsibleTowerJob < ::OrchestrationStack
+  require_dependency 'manageiq/providers/ansible_tower/configuration_manager/ansible_tower_job/status'
+
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::ConfigurationManager"
+  belongs_to :job_template, :foreign_key => :orchestration_template_id, :class_name => "ConfigurationScript"
+
+  #
+  # Allowed options are
+  #   :limit      => String
```

Comment:
@bzwei 
Is the limit the ip address of the server or the name of an inventory group